components/profile_popover/index.ts

```diff
@@ -67,9 +89,15 @@ function makeMapStateToProps() {
         const lastActivityTimestamp = getLastActivityForUserId(state, userId);
         const timestampUnits = getLastActiveTimestampUnits(state, userId);
         const enableLastActiveTime = displayLastActiveLabel(state, userId);
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        const isCallsEnabled = Boolean(state.plugins.plugins[suitePluginIds.calls]) && Boolean(state['plugins-' + suitePluginIds.calls]);
```

Comment:
I had to add ignore comment as `plugins` is not very well typed in `GlobalState`